### ISSUE TITLE
Auto-restart on /stop.

### ISSUE THREAD

**Issue by Mizening** [Minehut/Meta#965]
### Feature request

**Checklist**
- [ ] I have read and agree to the README (found on https://github.com/Minehut/Meta)
- [ ] This is not a bug report, support request or plugin request
- [ ] I have searched the Issue Tracker to make sure this is not a duplicate issue
- [ ] This issue does not contain any sensitive information 

**Platform**
Panel

**Describe the feature**
Ability to auto-restart server on /stop like Pterodactyl.

**How the feature is useful**
For minigame networks with sub-servers restarting automatically.

---
**Use 👍 and 👎 reactions to indicate you agree or disagree with this suggestion! Feel free to discuss the idea in the comments.**

**Comment by Tringlle** [comment]
Technically this is possible by yourself, probably disabled by minehut, if not should be (@drtshock). Create a file called `t.sh` and put `java -jar server.jar` inside it. Save it & create it. Go to spigot.yml and change `auto-restart-on-stop` to true (I think its called that). Also change `script-name` or whatever to `t.sh` or the name of the file you created. I have no idea if this will work or not, but its there.

**Comment by CoreyShupe** [comment]
It'll break the container's functionality if you try to do something like this. We could probably add something like this through the panel. I like the idea.

**Comment by Mizening** [comment]
Yeah, I have a UHC Meetup server and well I have to be on for them to be able to play (since the server reboots at the end of the game to clear world) This would definitely help out :D

**Comment by CoreyShupe** [comment]
> Yeah, I have a UHC Meetup server and well I have to be on for them to be able to play (since the server reboots at the end of the game to clear world) This would definitely help out :D

As a temporary solution you can probably have your server hibernate itself to reset if you have a stateless server, just load a bunch of stuff into RAM and crash your server.

I know it's not _the best_ option, but it is an option.

**Comment by CoreyShupe** [comment]
You can use:
```bash
echo {1..1000000000}
``` as your "restart-script" and it will force-hibernate your server (by crashing it), unsure how well that works for your situation.

**Comment by Mizening** [comment]
> > Yeah, I have a UHC Meetup server and well I have to be on for them to be able to play (since the server reboots at the end of the game to clear world) This would definitely help out :D
> 
> As a temporary solution you can probably have your server hibernate itself to reset if you have a stateless server, just load a bunch of stuff into RAM and crash your server.
> 
> I know it's not _the best_ option, but it is an option. (people can then start it from the lobby)

Yeah no it's apart of a network sadly so this probably wouldn't work.

**Comment by CoreyShupe** [comment]
> > > Yeah, I have a UHC Meetup server and well I have to be on for them to be able to play (since the server reboots at the end of the game to clear world) This would definitely help out :D
> > 
> > 
> > As a temporary solution you can probably have your server hibernate itself to reset if you have a stateless server, just load a bunch of stuff into RAM and crash your server.
> > I know it's not _the best_ option, but it is an option. (people can then start it from the lobby)
> 
> Yeah no it's apart of a network sadly so this probably wouldn't work.

Actually it would work better since minehut will repair the server and boot it back up. (If you only crash the sub servers)
